src/clipboard-action.js

```diff
@@ -64,15 +64,17 @@ class ClipboardAction {
         this.fakeElem.style[ isRTL ? 'right' : 'left' ] = '-9999px';
         // Move element to the same position vertically
         let yPosition = window.pageYOffset || document.documentElement.scrollTop;
-        this.fakeElem.addEventListener('focus', window.scrollTo(0, yPosition));
+        console.log("yPosition", yPosition)
+        // this.fakeElem.addEventListener('focus', window.scrollTo(0, yPosition));
         this.fakeElem.style.top = yPosition + 'px';
 
         this.fakeElem.setAttribute('readonly', '');
         this.fakeElem.value = this.text;
 
         document.body.appendChild(this.fakeElem);
-
-        this.selectedText = select(this.fakeElem);
+        this.fakeElem.select();
+        this.fakeElem.setSelectionRange(0, this.fakeElem.value.length);
+        this.selectedText = this.fakeElem.value;
```

Comment:
The `select` package already do this kind of logic https://github.com/zenorocha/select/blob/master/src/select.js#L9-L13. The difference is that it calls `focus()` instead of `select()`. Can you use the select package and submit a PR to it?